aggregation/simple/src/test/java/com/spotify/heroic/aggregation/simple/AggregationSerializationTest.java

```diff
@@ -1,10 +1,14 @@
 package com.spotify.heroic.aggregation.simple;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.spotify.heroic.aggregation.Aggregation;
 import com.spotify.heroic.aggregation.AggregationInstance;
+import com.spotify.heroic.aggregation.TDigestBucket;
```

Comment:
I don't see tests with TDigestBucket - are you planning to add them later?

Reply:
It is included  [here](https://github.com/spotify/heroic/pull/728/files#diff-caafa90d245b95c9bad6b90c9b7e78f3d1576b90e3a874276725aac27470c631R16)

Reply:
Then I guess you could remove the above import if it's not used, right?

Reply:
Now I see you removed the unused imports - next time pls add a note that you've addressed the comment and resolve it. Thank you.

Reply:
Hey @sergey I think, I misunderstood your concern then. Yes, that unused import was removed.  
So your question is about TDigestBucket sterilization test. Of course, I will push that with the next PR . This type of registration is required  https://github.com/spotify/heroic/blob/de3489066ce029137885014410aafd529c85ab2a/aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/Module.java#L78